src/PointwiseFunctions/GeneralRelativity/Surfaces/Python/Bindings.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+#include <pybind11/pybind11.h>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/Strahlkorper.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/Tags.hpp"
+#include "ParallelAlgorithms/ApparentHorizonFinder/HorizonAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Surfaces/Tags.hpp"
+#include "Utilities/ErrorHandling/SegfaultHandler.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace py = pybind11;
+
+namespace gr::surfaces::py_bindings {
+
+template <typename Frame>
+void bind_horizon_quantities_impl(py::module& m) {
+  static constexpr size_t Dim = 3;
+  m.def(
+      "horizon_quantities",
+      [](ylm::Strahlkorper<Frame> horizon,
+         tnsr::ii<DataVector, Dim, Frame> spatial_metric,
+         tnsr::II<DataVector, Dim, Frame> inv_spatial_metric,
```

Comment:
The spatial metric is in the output anyway because we load it as initial data. I could drop the `inv_spatial_metric` from the volume data and compute it here. But the horizon finder needs it, so it would have to compute the inverse in every iteration of the horizon finder. Not sure what's best, so I think just keep as is for now?

Reply:
I'm just worried we are making assumptions about what will be available when we don't necessarily know. Yes for the ID solve we'll have the spatial metric, but what about for a BNS run where somebody wants to see if a horizon formed? Or a supernova bounce? What about taking both as optionals? Then if both are available, you can just use both, but if one or the other are specified then you'll have to compute the inverse (you'd be doing it in volume data anyways)

Reply:
Can we worry about these applications later? You can always use `spectre transform-vol` to compute the inverse and other quantities. I could do the optionals but that adds quite a lot of logic that I'm not sure is needed and I think is premature optimization.

Reply:
Yeah we can worry about them later. I do think people will want to have the option though.

Reply:
Yes, we want to find horizons in evolution output data. Note that we also have to output ExtrinsicCurvature, spatial Christoffels, and the spatial Ricci tensor, which we probably don't do atm. So we either have to output those (lots of data), or postprocess with `transform-vol` (possible, maybe best solution and works already), or build the postprocessing into the horizon finder routines (possible, but can be done when we need it), or have more flexible horizon finder triggers in the evolution  (hopefully enabled by interpolation framework changes, but doesn't help if you already have the simulation output).